src/endpoints/authentication/CMakeLists.txt

```diff
@@ -18,6 +19,7 @@ target_include_directories(
   ${IRODS_EXTERNALS_FULLPATH_BOOST}/include
   ${IRODS_EXTERNALS_FULLPATH_FMT}/include
   ${IRODS_EXTERNALS_FULLPATH_SPDLOG}/include
+  ${IRODS_EXTERNALS_FULLPATH_JWTCPP}/include
```

Comment:
Instead of using `IRODS_EXTERNALS_FULLPATH_JWTCPP`, add `jwt-cpp::jwt-cpp` to `target_link_libraries`. You may also be able to remove the `JWT_DISABLE_PICOJSON` (but probably not).

Reply:
The `JWT_DISABLE_PICOJSON` macro was necessary for reaching a successful build.

Looking at the externals instructions for building the jwt package, seems we need to always define the macro.

Should it be part of `IRODS_COMPILE_DEFINITIONS_PUBLIC/PRIVATE` variables defined by irods/irods or is there a better way to automate that?

Reply:
Nah, it makes sense to put it here if it's needed.

Reply:
Are there reasons to never define that macro?

Reply:
You wouldn't need to define it if it's passed down from `jwt-cpp::jwt-cpp`

Reply:
Nice. @MartinFlores751 Please try using that instead.

Reply:
I don't remember if it is or not. Regardless, this line should still be removed from `target_include_directories` and `jwt-cpp::jwt-cpp` and `nlohmann_json::nlohmann_json` added to `target_link_libraries`. My original commit does this. I'm not sure why it didn't make it over to this PR.

Reply:
The PR was rebased again. The changes you suggested should land again eventually.

Reply:
It should be put back in place now.